### PR TITLE
PDI-15038 Support MongoDB AuthenticationDatabase Parameter in MongoDB…

### PR DESCRIPTION
**Pull request by flatrixnet**
… Steps and Support the SCRAM-SHA-1 security

### CODE REVIEW

**Comment by lucboudreau** [comment]
Please rebase.

**Comment by flatrixnet** [comment]
Rebased

File: pom.xml [hunk 1]
```diff
@@ -5,11 +5,11 @@
   <parent>
     <groupId>org.pentaho</groupId>
     <artifactId>pentaho-ce-parent-pom</artifactId>
-    <version>1.0.5</version>
+    <version>1.1.1</version>
   </parent>
   <groupId>pentaho</groupId>
   <artifactId>pentaho-karaf</artifactId>
-  <version>7.0-SNAPSHOT</version>
+  <version>6.1-SNAPSHOT</version>
```
Comment (pamval) [inline pom.xml]: this is wrong. Version should be 7.0-SNAPSHOT

**Comment by pamval** [comment]
What are the actual changes here ? The diff is showing a lot of minor spacing changes but I can't easily figure out what were the actual changes

**Comment by flatrixnet** [comment]
This "auto" refactor changed more than I expected.
But was in the end the upgrade of the mongo DB driver from  2.13.0 to 2.13.3.
This new driver includes on the manifest the javax/crypto/spec/PBEKeySpec this solves the problem below 
http://jira.pentaho.com/browse/PDI-14596

OLD 
<mongo.java.driver.version>2.13.0</mongo.java.driver.version>
NEW
<mongo.java.driver.version>2.13.3</mongo.java.driver.version>

**Comment by pamval** [comment]
Done in #191

**Pull request closed without merging**